metakb/version.py

```diff
@@ -1,4 +1,4 @@
 """MetaKB version"""
 # REQ: EACH TIME VERSION IS UPDATED, MUST ALSO UPDATE LAST_UPDATED
 __version__ = "1.1.1"
```

Comment:
maybe too late to start doing this now but we probably should've been using a 'dev' or 'alpha' marker in the version numbering

Reply:
We could bump to `1.2.0-dev0`

Reply:
@jsstevenson let me know if you want 1.1.1, 1.2.0-dev0, or something else. I think the minor bump makes sense

Reply:
@korikuzma whatever works! I was just thinking that it's not really released yet (or is it???) so numbering might need to reflect that

Reply:
Makes sense, I'll switch to 1.2.0-dev0